src/v2/guide/render-function.md

```diff
@@ -230,23 +230,23 @@ Vue.component('anchored-heading', {
 })
 ```
 
-### Constraints
+### Contraintes
 
-#### VNodes Must Be Unique
+#### Les VNodes doivent être unique
 
-All VNodes in the component tree must be unique. That means the following render function is invalid:
+Tous les VNodes dans l'arbre des composants doivent être unique. Cela signifie que les fonctions suivantes sont invalides :
 
 ``` js
 render: function (createElement) {
   var myParagraphVNode = createElement('p', 'hi')
   return createElement('div', [
-    // Yikes - duplicate VNodes!
+    // Erf - VNodes dupliqués !
     myParagraphVNode, myParagraphVNode
   ])
 }
 ```
 
-If you really want to duplicate the same element/component many times, you can do so with a factory function. For example, the following render function is a perfectly valid way of rendering 20 identical paragraphs:
+Si vous souhaitez réellement dupliquer le même élément/composant plusieurs fois, vous pouvez le faire avec une fonction de fabrique. Par exemple, la fonction de rendu suivante est parfaitement valide pour faire le rendu de 20 paragraphes identiques :
```

Comment:
Je préfère « de fabrique » pour concorder avec les autres itérations du genre même si ça ne me gène pas de voir écrit fonction fabrique plutôt que fonction de fabrique.

Comment je voyais le truc :

« factory function » : fonction de fabrique
« callback function » : fonction de rappel
« render function » : fonction de rendu

Mais effectivement « fonction fabrique » peut se rapprocher de « fonction constructeur » par exemple.

Je le change pour coller avec ton usage dans l'API j'imagine :)